### ISSUE TITLE
Function Argument Hints Do Not Stick

### ISSUE THREAD

**Issue by xstodeepak** [DaxStudio/DaxStudio#5]
Function argument hints do not stick once the first argument is entered. The user does not have any idea about the function arguments once he/she starts typing in the arguments. The hints should stick like in Excel making it very easy to author a function. It's a very basic requirement in any IntelliSense. Hope it gets fixed in the next release.

**Comment by dgosbell** [comment]
yes, the current function insights tooltips are only triggered on an open bracket character (

We would love to enhance this functionality and probably will as some point in the future, but doing it properly is going to take the implementation of a full DAX parser rather than the simple tokenizer that we use at the moment which is a massive piece of work, so it's unlikely to change soon.

**Comment by reubster** [comment]
Just wanted to vote for this. My brain has extremely limited short term memory capacity. I need constant reminding of function parameters, especially while typing the function.  

I recently started using VS Code, and I'm loving their model for viewing definitions of functions. Something like this would be fantastic in Dax Studio.
